tests/conftest.py

```diff
@@ -407,3 +407,22 @@ def teardown_bz_1696862():
         ansible_module.command(
             'mv {} {}'.format(contacted.values()[0]['backup'], satellite_answer_file))
     request.addfinalizer(teardown_bz_1696862)
+
+
+@pytest.fixture(scope='function')
+def setup_bz_1632768(request, ansible_module):
```

Comment:
It would be good to make generic like setup_hammer_defaults, same fixture can help us at other places too.